### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by cgiroux86**
+reviewer:@chg1017
+@girouxchris1017
+bug

---
This change is [Reviewable](https://reviewable-dev-chris-default-rtdb.firebaseapp.com/reviews/cgiroux86/test/521)

### CODE REVIEW

**Review by cgiroux86 — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/cgiroux86/test/521#-:-MwXPQLz3V-srQPB6Lmp:b-p9jurp)* status: 0 of 1 files reviewed, all discussions resolved (waiting on @chg1017)

**Review by cgiroux86 — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/cgiroux86/test/521#-:-MwXPzP42lVwAKGKbd7D:b2m1y8p)* status: 0 of 1 files reviewed, all discussions resolved (waiting on @chg1017)

**Comment by cgiroux86** [comment]
___
*[added.py, line 2 at r1](http://localhost:8080/reviews/cgiroux86/test/521#-Mx6C2Dy8loz4CHGasxT:-Mx6C2Dy8loz4CHGasxU:b-mhdmyn) ([raw file](https://github.com/cgiroux86/test/blob/d4a44dbd15035e2a4d1e2b6232aecf2cb18cd112/added.py#L2)):*
> ```Python
> print('This is an added file!!!!');
> print('TEST!!!!');
> ```

omment

**Review by cgiroux86 — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/cgiroux86/test/521#-:-MwYYNiJ1nD0tw-A04Az:bw8h5bf)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @chg1017)

___
*[added.py, line 1 at r1](http://localhost:8080/reviews/cgiroux86/test/521#-Mx6C4FP5-zyIah97dar:-Mx6C4FP5-zyIah97das:b7r5xui) ([raw file](https://github.com/cgiroux86/test/blob/d4a44dbd15035e2a4d1e2b6232aecf2cb18cd112/added.py#L1)):*
> ```Python
> print('This is an added file!!!!');
> ```

Comment

**Review by girouxchris1017 — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](http://localhost:8080/reviews/cgiroux86/test/521)* status: all files reviewed (commit messages unreviewed), 3 unresolved discussions (waiting on @cgiroux86 and @chg1017)

___
*[-- commits, line 2 at r1](http://localhost:8080/reviews/cgiroux86/test/521#-MxBrB2V86pyjXv_WQAs):*
Test
___
*[added.py, line 1 at r1](http://localhost:8080/reviews/cgiroux86/test/521#-Mx6C4FP5-zyIah97dar:-Mx6rs5iCu-5K_xqPSCZ:bb74njr) ([raw file](https://github.com/cgiroux86/test/blob/d4a44dbd15035e2a4d1e2b6232aecf2cb18cd112/added.py#L1)):*
<details><summary><i>Previously, cgiroux86 (Chris Giroux) wrote…</i></summary><blockquote>

Comment
</blockquote></details>

Test
___
*[added.py, line 2 at r1](http://localhost:8080/reviews/cgiroux86/test/521#-Mx6C2Dy8loz4CHGasxT:-Mx6rua8Dd7Q7nklQPcz:bb74njr) ([raw file](https://github.com/cgiroux86/test/blob/d4a44dbd15035e2a4d1e2b6232aecf2cb18cd112/added.py#L2)):*
<details><summary><i>Previously, cgiroux86 (Chris Giroux) wrote…</i></summary><blockquote>

omment
</blockquote></details>

Test

**Review by cgiroux86 — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/cgiroux86/test/521)* status: all files reviewed (commit messages unreviewed), 2 unresolved discussions (waiting on @cgiroux86, @chg1017, and @girouxchris1017)

___
*[-- commits, line 2 at r1](http://localhost:8080/reviews/cgiroux86/test/521#-MxBrB2V86pyjXv_WQAs):*
<details><summary><i>Previously, girouxchris1017 wrote…</i></summary><blockquote>

Test
</blockquote></details>

Done.
___
*[added.py, line 1 at r1](http://localhost:8080/reviews/cgiroux86/test/521#-Mx6C4FP5-zyIah97dar:-MxCRLjO45bcqmlijLjA:b-896fix) ([raw file](https://github.com/cgiroux86/test/blob/d4a44dbd15035e2a4d1e2b6232aecf2cb18cd112/added.py#L1)):*
<details><summary><i>Previously, girouxchris1017 wrote…</i></summary><blockquote>

Test
</blockquote></details>

Done.

**Review by cgiroux86 — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/cgiroux86/test/521)* status: all files reviewed (commit messages unreviewed), 1 unresolved discussion (waiting on @chg1017 and @girouxchris1017)

___
*[added.py, line 2 at r1](http://localhost:8080/reviews/cgiroux86/test/521#-Mx6C2Dy8loz4CHGasxT:-MxCRRsX1ihsrY0ErhDX:b-896fix) ([raw file](https://github.com/cgiroux86/test/blob/d4a44dbd15035e2a4d1e2b6232aecf2cb18cd112/added.py#L2)):*
<details><summary><i>Previously, girouxchris1017 wrote…</i></summary><blockquote>

Test
</blockquote></details>

Done.

**Review by girouxchris1017 — CHANGES_REQUESTED** [review]
> *[Reviewable](http://localhost:8080/reviews/cgiroux86/test/521)* status: all files reviewed (commit messages unreviewed), 1 unresolved discussion (waiting on @cgiroux86 and @chg1017)

___
*[-- commits, line 2 at r1](http://localhost:8080/reviews/cgiroux86/test/521#-MxBrB2V86pyjXv_WQAs):*
<details><summary><i>Previously, cgiroux86 (Chris Giroux) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

Not done?
___
*[added.py, line 1 at r1](http://localhost:8080/reviews/cgiroux86/test/521#-Mx6C4FP5-zyIah97dar:-MxCReKO3IksqwPa4eGI:bb7110x) ([raw file](https://github.com/cgiroux86/test/blob/d4a44dbd15035e2a4d1e2b6232aecf2cb18cd112/added.py#L1)):*
<details><summary><i>Previously, cgiroux86 (Chris Giroux) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

Not?
___
*[added.py, line 2 at r1](http://localhost:8080/reviews/cgiroux86/test/521#-Mx6C2Dy8loz4CHGasxT:-MxCRcISBcoCfGe5xJW-:bb7110x) ([raw file](https://github.com/cgiroux86/test/blob/d4a44dbd15035e2a4d1e2b6232aecf2cb18cd112/added.py#L2)):*
<details><summary><i>Previously, cgiroux86 (Chris Giroux) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

Not?

**Comment by cgiroux86** [comment]
TESTING

**Comment by cgiroux86** [comment]
COMMENT

**Comment by cgiroux86** [comment]
This is a test

**Review by cgiroux86 — COMMENTED** [review]
:lgtm:

> *[Reviewable](http://localhost:8080/reviews/cgiroux86/test/521#-:-MxL-7Dy60p1UVPxXRMg:bnfp4nl)* status: all files reviewed (commit messages unreviewed), 1 unresolved discussion (waiting on @cgiroux86 and @chg1017)

**Comment by cgiroux86** [comment]
*[a discussion](http://localhost:8080/reviews/cgiroux86/test/521#-MxL04_j98872t-JUWu3:-MwbVgaB2UXSnUFezJug:beu4iyi) (no related file):*


---

**Comment by cgiroux86** [comment]
TESTING!

File: added.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 print('This is an added file!!!!');
-print('TEST!!!');
\ No newline at end of file
+print('TEST!!!!');
```
Comment (cgiroux86) [inline added.py:2]: Comment!

**Review by cgiroux86 — COMMENTED** [review]
TEST

> *[Reviewable](http://localhost:8080/reviews/cgiroux86/test/521#-:-MxLBs64BhIqb1rJacCh:bb73z13)* status: all files reviewed (commit messages unreviewed), 1 unresolved discussion (waiting on @cgiroux86 and @chg1017)

**Review by cgiroux86 — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/cgiroux86/test/521)* status: all files reviewed (commit messages unreviewed), 1 unresolved discussion (waiting on @chg1017 and @girouxchris1017)

___
*[-- commits, line 2 at r1](http://localhost:8080/reviews/cgiroux86/test/521#-MxBrB2V86pyjXv_WQAs):*
<details><summary><i>Previously, girouxchris1017 wrote…</i></summary><blockquote>

Not done?
</blockquote></details>

COMMENT!

File: added.py (hunk 1 above)
Comment (girouxchris1017) [inline added.py:2]: Comment!

File: added.py [hunk 2]
```diff
@@ -1,2 +1,2 @@
 print('This is an added file!!!!');
-print('TEST!!!');
```
Comment (girouxchris1017) [inline added.py:2]: Comment!

**Review by cgiroux86 — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/cgiroux86/test/521)* status: all files reviewed (commit messages unreviewed), 3 unresolved discussions (waiting on @cgiroux86, @chg1017, and @girouxchris1017)

___
*[-- commits, line 1 at r1](http://localhost:8080/reviews/cgiroux86/test/521#-MxLJNx10LTtG9iylErk):*
Comment!

**Review by cgiroux86 — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/cgiroux86/test/521)* status: all files reviewed (commit messages unreviewed), 4 unresolved discussions (waiting on @cgiroux86, @chg1017, and @girouxchris1017)

___
*[added.py, line 1 at r1](http://localhost:8080/reviews/cgiroux86/test/521#-MxLJXt9CMD6GJZRklKX:-MxLJXt9CMD6GJZRklKY:b-kbdio7) ([raw file](https://github.com/cgiroux86/test/blob/d4a44dbd15035e2a4d1e2b6232aecf2cb18cd112/added.py#L1)):*
> ```Python
> print('This is an added file!!!!');
> ```

COMMENTING!

File: added.py (hunk 2 above)
Comment (cgiroux86) [inline added.py:2]: COMMENTING!

File: added.py (hunk 1 above)
Comment (cgiroux86) [inline added.py:2]: Comment!

**Comment by cgiroux86** [comment]
:lgtm:

**Comment by cgiroux86** [comment]
:lgtm:

**Comment by cgiroux86** [comment]
This is a test

**Review by cgiroux86 — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/cgiroux86/test/521)* status: all files reviewed (commit messages unreviewed), 4 unresolved discussions (waiting on @chg1017 and @girouxchris1017)

*[a discussion](http://localhost:8080/reviews/cgiroux86/test/521#-MxevBnP8sOWYYH3NMMF:-MxejeEy5vT_58tP5w_s:bmrzlax) (no related file):*


---

**Comment by cgiroux86** [comment]
T

**Comment by cgiroux86** [comment]
This is a test draft

**Comment by cgiroux86** [comment]
this

**Comment by cgiroux86** [comment]
This is a test draft

**Comment by cgiroux86** [comment]
Test draft

**Comment by cgiroux86** [comment]
Test draft

**Comment by cgiroux86** [comment]
Test draft

**Comment by cgiroux86** [comment]
TEST

**Comment by cgiroux86** [comment]
testing!

**Pull request merged**